web/src/components/Trace/TraceError/TraceError.tsx

```diff
@@ -0,0 +1,35 @@
+import {CloseCircleFilled} from '@ant-design/icons';
+import {Button, Typography} from 'antd';
+import React from 'react';
+import {ITestRunResult} from 'types/TestRunResult.types';
+import {useRunTestMutation} from 'redux/apis/Test.api';
+import * as S from '../Trace.styled';
+
+interface IProps {
+  displayError: boolean;
+  onDismissTrace: () => void;
+  testId: string;
+
+  onRunTest(result: ITestRunResult): void;
+}
+
+export const TraceError = ({displayError, onDismissTrace, onRunTest, testId}: IProps): JSX.Element | null => {
```

Comment:
I made some changes to this, please check https://github.com/kubeshop/tracetest/pull/440